crates/consensus/src/transaction/deposit.rs

```diff
@@ -72,9 +72,9 @@ impl TxDeposit {
             to: Decodable::decode(buf)?,
             mint: if *buf.first().ok_or(DecodeError::InputTooShort)? == EMPTY_STRING_CODE {
                 buf.advance(1);
-                None
+                0
             } else {
-                Some(Decodable::decode(buf)?)
+                Decodable::decode(buf)?
             },
```

Comment:
I believe we can now also simplify this to just `Decodable::decode(buf)?`

because `EMPTY_STRING_CODE` is the same as rlp(0)